scripts/build_gst.sh

```diff
@@ -267,9 +267,9 @@ EOF
 fi
 
 GST_BUILD_DIR=builddir
-meson setup ${GST_MESON_OPTIONS[@]} $GST_BUILD_DIR
+meson setup "$GST_BUILD_DIR" "${GST_MESON_OPTIONS[@]}"
```

Comment:
Does this work? The help says `meson setup [args] [builddir] [sourcedir]"

Reply:
The CI is green, maybe we should add tests to ensure the gst installation ?

Reply:
From the logs, meson receives those parameters correctly, so that works.

Regarding tests, we do have some tests to ensure plugins are installed with no missing dynamic libraries in the `inspect_gst_plugins.sh` script.

Reply:
That's true, forgot about those

Reply:
Ha... The documents I read got the the parameters in the other order... I didnt' double check with the version used so maybe I am wrong on this.
It was building thus

Reply:
@joaoantoniocardoso meson docs is a bit confusing about this but on current version the syntax is this one.
 The documentation example also state this on hints : https://mesonbuild.com/Running-Meson.html#configuring-the-build-directory 
So it should be good